boilerplate/bin/postInstall

```diff
@@ -15,8 +15,9 @@ if (["darwin", "linux"].includes(process.platform)) {
 }
 
 // On iOS, make sure CocoaPods are installed
+// set env var IGNITE_SKIP_POD_INSTALL to any value to skip
 if (process.platform === "darwin") {
-  run('if [ -d "ios" ]; then cd ios && pod install && cd -; fi')
+  run('if [ -d "ios" ]; then cd ios && npx pod-install && cd -; fi')
```

Comment:
`npx pod-install` now here over just plain `pod install`. This helps check for updating the pod repo etc if things are out of sync

Reply:
Oh nice I forgot that it did that, good call!